### ISSUE TITLE
Latest nightly breaks build

### ISSUE THREAD

**Issue by berkus** [leshow/iron-tera#2]
```
   Compiling iron-tera v0.4.0
   Compiling iron-csrf v0.2.4
error[E0119]: conflicting implementations of trait `std::convert::TryFrom<_>` for type `TemplateMode`:
   --> /Users/berkus/.cargo/registry/src/github.com-1ecc6299db9ec823/iron-tera-0.4.0/src/lib.rs:178:1
    |
178 | / impl<S> TryFrom<S> for TemplateMode
179 | | where
180 | |     S: Serialize,
181 | | {
...   |
185 | |     }
186 | | }
    | |_^
    |
    = note: conflicting implementation in crate `core`

error: aborting due to previous error

error: Could not compile `iron-tera`.
```

**Comment by leshow** [comment]
Sorry, I wasn't getting notifications on this for some reason.

I've just built on both nightly and stable without issue. Are you still having a problem?

**Comment by leshow** [comment]
Nevermind, I see it. Thanks for reporting

**Comment by leshow** [comment]
This is closed in 0.5.0, I've included examples for nightly & stable in the repo. Thank you

**Comment by berkus** [comment]
Excellent, thank YOU!
